back-end/schemas/schemas.sql

```diff
@@ -107,6 +112,8 @@ CREATE TABLE IF NOT EXISTS Post (
     post VARCHAR(255) NULL,
     thumbs_up INT NULL,
     user_id INT UNIQUE NOT NULL,
+    create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
+    updata_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
created_at would be a better name for the column, also since there is a default value it will never be null so you will have to remove the null

updated_at would be a better name for the column, and it is fine if it didn't have a default value.